src/nycdb/verify.py

```diff
@@ -5,6 +5,7 @@
     'pluto_18v1': {'pluto_18v1': 800_000},
     'pluto_18v2': {'pluto_18v2': 800_000},
     'pluto_19v1': {'pluto_19v1': 800_000},
+    'pluto_19v1': {'pluto_19v2': 800_000},
```

Comment:
typo -- should be pluto_19v2

Reply:
oops! ok just fixed, thanks for catching